CrazyCanvas/Source/States/SandboxState.cpp

```diff
@@ -172,7 +172,7 @@ void SandboxState::Init()
 
 		for (uint32 y = 0; y < gridRadius; y++)
 		{
-			const float32 roughness = y / float32(gridRadius - 1);
+			const float32 roughness = y / float32(gridRadius - 1) + 0.01f;
```

Comment:
just curious what does this small float apply

Reply:
This float increases the roughness on the lowest row of balls to go from perfect mirrors to just a little rough. This was to test the IBL in the IBL-branch, so I do not know why this is not in master already.